src/eXpansion/Bundle/VoteManager/Plugins/Votes/RestartMapVote.php

```diff
@@ -47,6 +51,7 @@ public function __construct(
 
         $this->jukebox = $jukebox;
         $this->mapStorage = $mapStorage;
+        $this->map = $mapStorage->getCurrentMap();
```

Comment:
Not proper place, this will restart always the map on which the contoller started.

This is a plugin a single instance is created at controller start and same is used later on. Need to add MapDataProvider and do this on beginMap & on setStatus true